dev/tools/lib/roll_dev.dart

```diff
@@ -114,12 +143,12 @@ void main(List<String> args) {
   // Publish the archive before pushing the tag so that if something fails in
   // the publish step, we can clean up.
   try {
-    new ArchivePublisher(hash, version, Channel.dev)..publishArchive();
+    publisher.publishArchive();
   } on ArchivePublisherException catch (e) {
     print('Archive publishing failed.\n$e');
-    runGit('tag -d $version', 'remove the tag that was not published');
+    runGit('tag -d v$version', 'remove the tag that was not published');
```

Comment:
Whoa, good catch.

Reply:
I'd actually forgotten about all of them, then when I submitted my PR to fix it it merge-conflicted with yours, and I missed this one when merging.